.github/workflows/inspektor-gadget.yml

```diff
@@ -236,7 +236,7 @@ jobs:
         file: /home/runner/work/inspektor-gadget/inspektor-gadget/Dockerfiles/gadget-${{ matrix.type }}.Dockerfile
         build-args: |
           ENABLE_BTFGEN=true
-        outputs: type=registry,name=${{ steps.set-repo-determine-image-tag.outputs.image-tag.outputs.container-repo }},push=true,push-by-digest=true
+        outputs: type=registry,name=${{ steps.set-repo-determine-image-tag.outputs.container-repo }}:${{ steps.set-repo-determine-image-tag.outputs.image-tag }},push=true,push-by-digest=true
```

Comment:
Did you try to merge this change in a fork? The CI of https://github.com/kinvolk/inspektor-gadget/pull/921 was passing on the PR but failed after the merge.